src/main/java/org/example/dao/PointDAOImpl.java

```diff
@@ -0,0 +1,80 @@
+package org.example.dao;
+
+import org.example.model.Point;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class PointDAOImpl implements PointDAO {
+    private Connection connection;
+
+    public PointDAOImpl() {
+        try {
+            connection = DriverManager.getConnection("jdbc:mysql://localhost:3306/DB", "username", "password");
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    @Override
+    public void insertPoint(Point point) {
```

Comment:
This looks like a JDBC implementation. I thought we were doing MyBatis. I mean, it's fine. We can switch between them.

Reply:
yes,it's just for now